airbyte-integrations/connectors/source-bigquery/unit_tests/integration/mocked.py

```diff
@@ -0,0 +1,41 @@
+from typing import Mapping, Set
+from integration.request_builder import BigqueryRequestBuilder
+from integration.response_builder import BigqueryResponseBuilder
+from integration.bq_query_builder import build_query
+
+
+def mock_discover_calls(http_mocker, tables: Mapping[str, Mapping[str, Set[str]]]) -> None:
```

Comment:
Can we add some docstring about what `tables` represent with an example? Without seeing usage, it is quite cryptic

Reply:
Sure.